bower.json

```diff
@@ -18,7 +18,7 @@
     "patternfly": "3.21.0",
     "jquery": "2.1.4",
     "lodash": "3.10.1",
-    "origin-web-common": "^0.0.10"
+    "origin-web-common": "~0.x"
```

Comment:
> Should this just be
> 
> "origin-web-common": "0.0.x"

I initially tried that,  it doesn't seem to work.  Maybe something with '0.0'? '1.0.x' works as expected.  I've tried it here: https://semver.npmjs.com/